lib/addons/index.ts

```diff
@@ -1,66 +0,0 @@
-
-export * from './ack';
```

Comment:
WHy is this file removed, this will break the whole functioning of addons with blueprints. You have to remove only your line.

Reply:
I reverted back this file deletion